src/annotator/integrations/html.ts

```diff
@@ -53,9 +81,11 @@ export class HTMLIntegration extends TinyEmitter implements Integration {
   constructor({
     features,
     container = document.body,
+    sideBySideOptions = {},
   }: {
     features: FeatureFlags;
     container?: HTMLElement;
+    sideBySideOptions?: Partial<SideBySideOptions>;
```

Comment:
I think this should be `SideBySideOptions` rather than `Partial<SideBySideOptions>`. `Partial<T>` makes every field optional, and that might not be what we want. For example, I think it makes sense to require the `mode` field at least.

Reply:
The idea was that it was possible to provide only some of the properties for side by side (in the eventual case we have more than one).

However, I'm now half-way working on how the config is read and parsed before getting here, and that logic is already taking care of defining default values, so this object can probably set as non-partial as you suggest.